client/.github/workflows/comment.md

```diff
@@ -1,6 +0,0 @@
-## Bundle size difference
```

Comment:
I thought that this file was required by the bundle size action?
```
- name: Comment
        uses: NejcZdovc/comment-pr@v1
        with:
          file: "comment.md"
```

Reply:
The same file was already in `.github/workflows/comment.md` thats why it doesn't show up in the PR

Reply:
ah. oops